app/src/sass/_lists.scss

```diff
@@ -0,0 +1,20 @@
+.bubble-list,
+.link-list {
```

Comment:
Probably not a good idea to reappropriate a common data structure. But it's to display the links in the footer.